docs/about/compatibility/precission-support.md

```diff
@@ -0,0 +1,128 @@
+|:-------------|:-----:|:------:|:-----:|:-----:|:-------------:|:-------------:|:-------:|:--------:|:-------------:|:-------:|:-------:|
+| Mi100        | ❌    |  ❌   | ❌    | ❌   | ❌           | ❌           | ❌      | ❌       | ❌           | ❌      | ❌     |
+| Mi200 series | ❌    |  ❌   | ❌    | ❌   | ❌           | ❌           | ✅      | ✅       | ❌           | ✅      | ✅     |
+| Mi300 series | ✅    |  ❌   | ❌    | ❌   | ✅           | ✅           | ✅      | ✅       | ✅           | ✅      | ✅     |
+
+## Hardware accelerated global memory atomic operations
+
+| Data types   | int8  | int16  | int32 | float8 (E4M3) | float8 (E5M2) | float16 | bfloat16 | tensorfloat32 | float32 | float64 |
+|:-------------|:-----:|:------:|:-----:|:-------------:|:-------------:|:-------:|:--------:|:-------------:|:-------:|:-------:|
+| Mi100        | ❌    |  ❌   | ✅    | ❌           | ❌           | ✅      | ✔        | ❌           | ✅      | ✔      |
+| Mi200 series | ❌    |  ❌   | ✅    | ❌           | ❌           | ✅      | ✔        | ❌           | ✅      | ✅     |
+| Mi300 series | ❌    |  ❌   | ✅    | ❌           | ❌           | ✅      | ✔        | ❌           | ✅      | ✅     |
+
+# Data Type support in ROCm Libraries
+
+ROCm™ components support for int8, float8 (E4M3), float8 (E5M2), int16, float16, bfloat16, int32, tensorfloat32,
+float32, int64 and float64 is listed in the following tables.
+
+## Libraries input/output type support
+
+The ROCm™ components support of specific input and output data type is listed in the following two tables. For detailed description open the library data type support page.
+
+<!---
+TODO: 
+- Check MIGraphX
+- Get confirmation on rocThrust, hipCUB and rocPRIM
+-->
+
+### Integer
+
+| Library input/output integer data types                               | int8  | int16   | int32 | int64   |
+|:----------------------------------------------------------------------|:-----:|:-------:|:-----:|:-------:|
+| hipSPARSELt ({doc}`details<hipsparselt:reference/data-type-support>`) | ✅/✅ | ❌/❌ | ❌/❌| ❌/❌  |
+| rocRAND ({doc}`details<rocrand:data-type-support>`)                   | -/✅  | -/✅   | -/✅  | -/✅   |
+| hipRAND ({doc}`details<hiprand:data-type-support>`)                   | -/⚠️  | -/✅   | -/✅  | -/✅   |
+| rocPRIM ({doc}`details<rocprim:data-type-support>`)                   | ✅/✅ | ✅/✅ | ✅/✅ | ✅/✅ |
+| hipCUB ({doc}`details<hipcub:data-type-support>`)                     | ✅/✅ | ✅/✅ | ✅/✅ | ✅/✅ |
+| rocThrust ({doc}`details<rocthrust:data-type-support>`)               | ✅/✅ | ✅/✅ | ✅/✅ | ✅/✅ |
+| MIGraphX                                                              | ✅/❌ | ❌/❌ | ❌/❌ | ❌/❌ |
+
+### Floating points
+
+| <div style="width:180px">Library input/output floating point data types</div> | float8 (E4M3) | float8 (E5M2) | float16 | bfloat16 | tensorfloat32 | float32 | float64 |
```

Comment:
"Library input/output floating point data types"

This is being rendered in a darker colour in the generated html file